library/eclipse-temurin

```diff
@@ -0,0 +1,28 @@
+# Eclipse Temurin OpenJDK images provided by the Eclipse Foundation
+
+Maintainers: George Adams <george.adams@microsoft.com> (@gdams_)
```

Comment:
Doh, we missed this in `library/adoptopenjdk` too -- this is a GitHub handle, not Twitter:

```suggestion
Maintainers: George Adams <george.adams@microsoft.com> (@gdams)
```